.eslintrc.js

```diff
@@ -29,6 +29,7 @@ module.exports = {
 		},
 		ecmaVersion: 2020,
 		sourceType: "module",
+		requireConfigFile: false,
```

Comment:
Is there a reason behind this configuration change? I'm not seeing a difference in results with the linter with or without this setting.

Reply:
On my computer it was giving me an error and wouldn't lint my files. Not sure why it was doing it but this is what fixed it. When I remove this line of code I get this error.

```
Parsing error: No Babel config file detected for /Users/tranb2/Sites/wpmedia/arc-themes-blocks/blocks/identity-block/components/login/index.jsx. Either disable config file checking with requireConfigFile: false, or configure Babel so that it can find the config files.eslint
```

Reply:
There's a babel config at the root of the project so that's strange it couldn't be found. Could you revert this line and push it back up (in both PRs please)? After that I can approve and merge both.

Reply:
@vgalatro Yeah, I was thinking the same thing. Maybe it is a VSCode issue. I updated both PR's. Thanks